fv3core/pace/fv3core/stencils/dyn_core.py

```diff
@@ -683,9 +682,6 @@ def __call__(
         dt_acoustic_substep = timestep / self.config.n_split
         dt2 = 0.5 * dt_acoustic_substep
         n_split = self.config.n_split
-        # TODO: When the namelist values are set to 0, use these instead:
-        # m_split = 1. + abs(dt_atmos)/real(k_split*n_split*abs(p_split))
-        # n_split = nint( real(n0split)/real(k_split*abs(p_split)) * stretch_fac + 0.5 )
```

Comment:
I couldn't tell what the comment means (what namelist values?), and the code has been dead for years, so I deleted it. We pivoted during that time to say we aren't going to include code for untested namelist options, and this seems to fall under that category.

Reply:
I think the idea here is that if `self.config.n_split` is not set you chose `n_split = nint( real(n0split)/real(k_split*abs(p_split)) * stretch_fac + 0.5 )`
and having an automatically determined n_split seems like a good idea, right?

Reply:
But what is n0split and p_split? why is it multiplied by stretch_fac and added to 0.5?

If you're sure this was the "default" value in the Fortran, I'd be fine with updating the comment to say that and leaving the commented code, but this would fall under "untested" code we'd normally delete.

Reply:
> But what is n0split and p_split? why is it multiplied by stretch_fac and added to 0.5?

I am totally the wrong person to answer that.  I'm ok with removing it if you prefer

Reply:
After our discussion I think I would still prefer removing it.